From f47d569311d51ec9ab5dad7452b43c1b5a8290b6 Mon Sep 17 00:00:00 2001
From: Glenn Strauss <gstrauss@gluelogic.com>
Date: Tue, 1 Dec 2020 16:41:58 -0500
Subject: [PATCH] [PATCH] [meson] do not search for lua versions

openwrt net/lighttpd/Makefile declares dependency on liblua, and not on
specific lua versions, so have meson.build look for 'lua' and not for
specific lua versions.

Signed-off-by: Glenn Strauss <gstrauss@gluelogic.com>
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/meson.build
+++ b/src/meson.build
@@ -377,7 +377,7 @@ endif
 liblua = []
 if get_option('with_lua')
 	found_lua = false
-	foreach l: ['lua5.4', 'lua-5.4', 'lua5.3', 'lua-5.3', 'lua5.2', 'lua-5.2', 'lua5.1', 'lua-5.1', 'lua']
+	foreach l: ['lua']
 		if not(found_lua)
 			liblua = dependency(l, required: false)
 			if (liblua.found())
